### PR TITLE
SL with latest JPA + with DNN Resolved + PR#52

### PR DESCRIPTION
**Pull request by gsaha009**
Changes : 
 - New JPA has been implemented
 - DNN for resolved JPA categories are in the Plotter
 - `mvaEvaluatorSL.py`has all DNN functions
 - new c++ function `jpa.h` is for 2 heavy high level variable
 - all SL signal xsecs are corrected in `analysis[era]SL_v7.yml`s

### CODE REVIEW

File: MachineLearning/HHMachineLearning/parameters_boosted.py [hunk 1]
```diff
@@ -0,0 +1,398 @@
...
+time = '0-06:00:00' # days-hh:mm:ss
+mem = '50000' # ram in MB
+tasks = 1 # Number of threads(as a string) (not parallel training for classic mode)
+cpus = 1
+gpus = 1
+workers = 7
+
+##################################  Naming ######################################
+# Physics Config #
+<<<<<<< HEAD:MachineLearning/HHMachineLearning/parameters_boosted.py
```
Comment (FlorianBury) [inline MachineLearning/HHMachineLearning/parameters_boosted.py:62]: Did you solve this conflict ?

File: MachineLearning/HHMachineLearning/parameters_boosted.py [hunk 2]
```diff
@@ -0,0 +1,398 @@
...
+#    'batch_size' : [10000], 
+#    'n_particles' : [16],
+#    'loss_function' : [grouped_loss] , #  [categorical_crossentropy]
+#}
+p = { 
+<<<<<<< HEAD:MachineLearning/HHMachineLearning/parameters_boosted.py
+    'lr' : [0.1], 
+=======
+    'lr' : [0.001], 
+>>>>>>> fd35d2cb6e4f979520e37c17eeacb7ff1354f9dd:MachineLearning/HHMachineLearning/parameters.py
```
Comment (FlorianBury) [inline MachineLearning/HHMachineLearning/parameters_boosted.py:186]: same

File: MachineLearning/HHMachineLearning/parameters_boosted.py (hunk 1 above)
Comment (gsaha009) [reply MachineLearning/HHMachineLearning/parameters_boosted.py:62]: Yes, I did. 
this `parametrers_boosted.py` is a new file with the features for boosted category

File: MachineLearning/HHMachineLearning/parameters_boosted.py (hunk 2 above)
Comment (gsaha009) [reply MachineLearning/HHMachineLearning/parameters_boosted.py:186]: I kept basically 2 files there.
`parameters.py` : as used in resolved 
`parameters_boosted.py` : for boosted case

File: MachineLearning/HHMachineLearning/parameters_boosted.py (hunk 1 above)
Comment (FlorianBury) [reply MachineLearning/HHMachineLearning/parameters_boosted.py:62]: I can still see the sign of the conflict in the files, either correct, commit and push to add on top of the changes, or I can do it in the next PR very soon

**Pull request merged**